src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
```

```suggestion
    transition: color 300ms;
```
Вказуй, яка властивість має анімуватись